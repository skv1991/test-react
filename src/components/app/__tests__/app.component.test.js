import React from 'react';
import App from '../component';
import {shallow} from "enzyme";

describe('App', () => {
  it('Рендеринг', () => {
    expect(shallow(<App />)).toMatchSnapshot();
  });
});
