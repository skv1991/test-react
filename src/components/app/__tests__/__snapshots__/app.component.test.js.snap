// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Рендеринг 1`] = `
<div
  className="app"
>
  <Post
    text="С чем-то"
    title="Что-то"
  />
</div>
`;
